app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageTypes.kt

```diff
@@ -90,8 +103,78 @@ internal fun MessageBody(
         StrikethroughExtension.builder().requireTwoTildes(true).build(),
         TablesExtension.create()
     )
+    text?.also {
+        MarkdownDocument(Parser.builder().extensions(extensions).build().parse(it) as Document, nodeData)
+    }
+    buttonList?.also {
+        MessageButtonsContent(
+            messageId = messageId,
+            buttonList = it,
+            onClick = onButtonClick,
+            pendingButton = pendingButton
+        )
+    }
+}
```

Comment:
I said it before, but maybe you forgot about it :P I Think instead of passing nullable stuff that sometimes are used and sometimes not, it makes more sense to extract this 
```

 buttonList?.also {
        MessageButtonsContent(
            messageId = messageId,
            buttonList = it,
            onClick = onButtonClick,
            pendingButton = pendingButton
        )
    }
```

to a seperate Composable